opentelemetry-sdk/src/resource/builder.rs

```diff
@@ -0,0 +1,174 @@
+use std::borrow::Cow;
+
+use opentelemetry::{KeyValue, Value};
+
+use super::{Resource, ResourceDetector, SERVICE_NAME};
+
+/// Builder to allow easy composition of a Resource
+#[derive(Debug)]
+pub struct ResourceBuilder {
+    resource: Resource,
+}
+
+impl Default for ResourceBuilder {
```

Comment:
we can remove the Default implementation  for the builder, so as to force users to always go via Resource to get hold of a builder.

Reply:
removed